### PR TITLE
Add multipicker renderer

### PR DESCRIPTION
**Pull request by Meroje**
This renderer wraps \Nos\Renderer_Item_Picker to use multiple on a single key, it can be used on a relation as an alternative to multiselect or hasmany renderers

It provides two options:
- limit can be used to allow a limited number of items
- picker_options is passed raw to Renderer_Item_Picker, see there for documentation

### CODE REVIEW

**Comment by cholletNovius** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ For now, renderers included are :
 - modelsearch
 - markdown wysiwyg editor
 - hasmany management
+- multipicker
```
Comment (shaoshiva) [inline README.md:13]: For consistency with the existing `item picker` I would have named it `multiple item picker`.

File: views/multipicker/inputs.view.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            'renderer_options' => $options['picker_options'],
+        )
+    );
+    echo '</div>';
+    $index++;
+}
+?>
+    <span data-next="<?= $index ?>" data-key="<?= $key ?>"></span>
+</div>
+<style>.multipicker { display: inline-block; float: left; }</style>
```
Comment (shaoshiva) [inline views/multipicker/inputs.view.php]: Missing newline at end of file.

File: views/multipicker/js.view.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                {
+                    var $newitem = $('<input name="'+ key +'[' + index + ']" class="item_picker" type="hidden" value="">');
+                    $container.append('<div class="multipicker multipicker_'+index+'"></div>').find('.multipicker_' + index).append($newitem);
+                    $newitem.nositempicker(item_options);
+                    $span.data('next', index + 1);
+                }
+            });
+        });
+    });
+</script>
```
Comment (shaoshiva) [inline views/multipicker/js.view.php]: Missing newline at end of file.

File: README.md (hunk 1 above)
Comment (Meroje) [reply README.md:13]: not a fan of this much verbosity

**Review by jguyomard — APPROVED** [review]

**Comment by Meroje** [comment]
Feel free to reopen this if you ever need it

**Pull request closed without merging**